api/HTMLElement.json

```diff
@@ -1079,54 +1079,6 @@
           }
         }
       },
-      "forceSpellCheck": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/forceSpellCheck",
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/forceSpellCheck will need to be removed.